examples/hello_services/ios/HelloServices/LocationProvider.h

```diff
@@ -0,0 +1,13 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Flutter/Flutter.h>
+
+@interface LocationProvider : NSObject <FlutterMessageListener>
+
+- (void) registerWithController:(FlutterViewController*) controller;
+
+- (NSString*)didReceiveString:(NSString*)message;
```

Comment:
`- (NSString*)didReceiveString:(NSString*)message;` is part of the `FlutterMessageListener` protocol. You do not need to re-declare it here.